.yamllint

```diff
@@ -8,6 +8,8 @@ ignore: |
   awx/ui/test/e2e/tests/smoke-vars.yml
   awx/ui/node_modules
   tools/docker-compose/_sources
+  # django template files
+  awx/api/templates/instance_install_bundle/**
```

Comment:
The yamllint checks _seem_ useful at minimum for this file

```diff
diff --git a/awx/api/templates/instance_install_bundle/inventory.yml b/awx/api/templates/instance_install_bundle/inventory.yml
index 1124cae88e..665e38863c 100644
--- a/awx/api/templates/instance_install_bundle/inventory.yml
+++ b/awx/api/templates/instance_install_bundle/inventory.yml
@@ -2,8 +2,8 @@
 all:
   hosts:
     remote-execution:
-      ansible_host: {{ instance.hostname }}
-      ansible_user: <username> # user provided
+      ansible_host: "{{ instance.hostname }}"
+      ansible_user: <username>  # user provided
       ansible_ssh_private_key_file: ~/.ssh/id_rsa
       receptor_verify: true
       receptor_tls: true
@@ -21,7 +21,7 @@ all:
       receptor_group: awx
       receptor_protocol: 'tcp'
       receptor_listener: true
-      receptor_port: {{ instance.listener_port }}
+      receptor_port: "{{ instance.listener_port }}"
       receptor_dependencies:
         - podman
         - crun
```

Those are good changes, and it would be good to have the checks turned on for these.

On the other hand, `awx/api/templates/instance_install_bundle/install_receptor.yml` does _not_ give useful feedback, but that's because it's a jinja2 template, not a yaml file. Again, not _bad_ feedback, because maybe the `*.j2` extension would be better to have on that file.

Reply:
current problem is that this is not actually a jinja2 template its django template thats why we are hesitant to add the .j2 extension